modules/core/src/main/java/com/illposed/osc/transport/OSCPort.java

```diff
@@ -70,6 +70,17 @@ protected OSCPort(
 		this(local, remote, serializerAndParserBuilder, NetworkProtocol.UDP);
 	}
 
+	protected OSCPort(
+		final Transport transport)
+	{
+		if (transport == null) {
+			throw new IllegalStateException(
```

Comment:
Would a [`NullPointerException`](https://docs.oracle.com/javase/8/docs/api/java/lang/NullPointerException.html) not be more appropriate?